Assets/Scripts/PowerDrivenMotor.cs

```diff
@@ -21,7 +18,7 @@ public class PowerDrivenMotor : Motor
     /// </summary>
     public float Speed
     {
-        get { return _speed; }
+        get { return _openLoopSpeed; }
```

Comment:
I had the same question asked by Isaac here. Do we always want `_openLoopSpeed`?

Reply:
As far as I can tell, I'm not sure if this property is even used anywhere, I mostly just put `_openLoopSpeed` since that's the analog of the previous `_speed` field. I figure since closed loop speed is handled internally within the class, everyone else should only be referring to its open loop speed anyway, right?

Reply:
Yeah, that makes sense. 👍🏻